src/api.js

```diff
@@ -41,8 +41,28 @@ const parseJob = rawJob => {
   };
 };
 
+const enqueueMessage = message => {
+  return axios.post('/messages', formatMessage(message));
+};
+
+const formatMessage = message => {
+  return {
+    actor_name: !message.actorName ? null : message.actorName,
```

Comment:
can you invert al the ternary operators ex: `message.actorName ? message.actorName : null ` make things more readable.